autorest/codegen/models/enum_schema.py

```diff
@@ -31,9 +32,10 @@ def from_yaml(cls, yaml_data: Dict[str, Any]) -> "EnumValue":
         :return: A created EnumValue
         :rtype: ~autorest.models.EnumValue
         """
+        value = yaml_data["value"]
         return cls(
             name=yaml_data["language"]["python"]["name"],
-            value=yaml_data["value"],
+            value=f"\"{value}\"" if is_string else value,
```

Comment:
don't do that, each schema has a `get_declaration` method to serialize correctly the value based to the actual type